CONTRIBUTING.md

```diff
@@ -213,44 +213,44 @@ Further use the **_OWASP WrongSecrets --> Lifecycle --> install_** step to load
 
 Open the **_WrongSecretsApplication_** by following the path **_main>java>org.owasp.wrongsecrets>WrongSecretApplication_**.
 
-![](images/open-application-6.1.png)
+![Click on the 'WrongSecretsApplication' file located at main > java > org.owasp.wrongsecrets > WrongSecretApplication](images/open-application-6.1.png)
 
 Press **_Shift+F10_** to run the application, this will open up the **_Run/Debug Configurations Menu._**
 
-![](images/run-application-6.2.png)
+![Click the 'Run' button or press Shift + F10](images/run-application-6.2.png)
 
 ### Step 8: Setting up Configurations.
 
 Select **_Edit configuration templates_** then select **_Application_** section.
 
-![](images/edit-config-7.1.png)
+![In the bottom left corner click on 'Edit configuration templates...'](images/edit-config-7.1.png)
 
 There under the **_Application_** section click on the button shown below.
 
-![](images/modify-options-7.2.png)
+![In the application section, click 'Modify options' to show below](images/modify-options-7.2.png)
 
 **_Select_** all the fields that are Selected in the below picture.
 
-![](images/select-options-7.3.png)
+![In the Run Options select: 'Enviroment variables' | 'Add VM options' | 'Shorten command line' | 'Specify classes and packages' | 'Open run/debug tool window when started'](images/select-options-7.3.png)
 
 **_Fill out_** all the fields as shown below.
 
-![](images/fill-fields-7.4.png)
+![Fill out the fields: | Build and run: org.owasp.wrongsecrets.WrongSecretsApplication : -Dserver.port=8080 - Dspring.profiles.active=local,without-vault. | Working directory: /Users/razr/workspace/owasp/wrongsecrets | Environment variables: K8S_ENV=docker| Packages and classes to include in covarege data: org.owasp.wrongsecrets.*| ](images/fill-fields-7.4.pn)
```

Comment:
Can you explain that the working directory depends on the enviroment setup of the user?

Reply:
It might be a good idea to describe which setting goes where :-)

Reply:
i'll do it :)